charts/opensearch-2.15.0/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Please remove network policy.  This is something the user can set up later if they choose.  Also, it is tailored to Verrazzano topology and this should be generic.

Reply:
It is removed.